File: MusicEventManagementSystem/MusicEventManagementSystem.Client/src/frontend/media-campaign/types/form/mediaVersion.ts
export interface MediaVersionCreateForm {
  versionFileName: string;
  fileType?: string;
  fileURL?: string;
  isFinalVersion: boolean;
  adId: number;
}

export interface MediaVersionUpdateForm {
  versionFileName?: string;
  fileType?: string;
  fileURL?: string;
  isFinalVersion?: boolean;
  adId?: number;
}

File: MusicEventManagementSystem/MusicEventManagementSystem.API/Models/MediaVersion.cs
namespace MusicEventManagementSystem.API.Models
{
    public class MediaVersion
    {
        public int MediaVersionId { get; set; }
        public int AdId { get; set; }
        public string? VersionFileName { get; set; }
        public string? FileType { get; set; }
        public string? FileURL { get; set; }
        public bool IsFinalVersion { get; set; }
        public DateTime CreatedAt { get; set; }
        public int? MediaTaskId { get; set; }

        // Navigation properties
        public virtual Ad Ad { get; set; } = null!;
        public virtual MediaTask? MediaTask { get; set; }
    }
}